doc/whats_new.rst

```diff
@@ -1,15 +1,17 @@
-:orphan:
-
 .. include:: links.inc
 
 .. _whats_new:
 
 .. currentmodule:: pyglmnet
 
-.. _current:
+=============
+Release notes
+=============
+
+.. _1_0_1:
```

Comment:
do we need this line?

Reply:
if we want to link from other files, this can be useful ...